testing/test_framework/src/firebase_test_framework.h

```diff
@@ -443,8 +455,12 @@ class FirebaseTest : public testing::Test {
   // Open a URL in a browser window, for testing only.
   static bool OpenUrlInBrowser(const char* url);
 
-  // Returns true if we can run tests that require interaction, false if not.
-  static bool IsUserInteractionAllowed();
+  // Returns true if we skip tests that require interaction, false if not.
+  static bool SkipUITest();
```

Comment:
Can we do this as a positive rather than a negative?
"ShouldRunUITests"
"ShouldRunNonUITests"